### PR TITLE
PMM-8079 Update operators

### PR DESCRIPTION
**Pull request by tiagomotasantos**
Shows instructions on how to update an operator when a new version is available

### CODE REVIEW

File: docs/using/dbaas.md [hunk 1]
```diff
@@ -77,6 +77,22 @@ The Kubernetes Cluster tab shows the status of operators.
 
 ![!](../_images/PMM_DBaaS_Kubernetes_Cluster_Operator_Status.png)
 
+### Kubernetes operator update
+
+When a new version of the operator is available the *Operators* column shows a message with this information. Cick the message to go to the operator release notes to find out more about the update.
```
Comment (jprukner) [inline docs/using/dbaas.md:82]: Click

**Review by jprukner — CHANGES_REQUESTED** [review]

File: docs/using/dbaas.md (hunk 1 above)
Comment (tiagomotasantos) [reply docs/using/dbaas.md]: Thanks done!

**Review by jprukner — APPROVED** [review]
:+1:

**Review by PaulJacobs-percona — APPROVED** [review]
I'll share a preview in the main comments.

**Comment by PaulJacobs-percona** [comment]
Preview https://60eff3b93ca8940008c5e743--pmm-doc.netlify.app/using/dbaas.html

**Review by PaulJacobs-percona — APPROVED** [review]

**Pull request merged**